### PR TITLE
feat: wait option

### PR DESCRIPTION
**Pull request by dberuben**
## Description

Waits until all Pods are in a ready state

## Motivation and Context

Make sure pods and others component are healthy

## How Has This Been Tested?

No 👶 

## Checklist:

- [X] My change requires a change to the documentation or CHANGELOG.
- [X] I have updated the CHANGELOG.md accordingly.
- [X] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Changes since 1.1.2
+# Changes since 1.1.3
+
+# 1.1.3
```
Comment (tuunit) [inline CHANGELOG.md:3]:
```suggestion
# Changes since 1.1.2
```

Please change back

**Review by tuunit — CHANGES_REQUESTED** [review]
Please change back the Changelog heading other than that everything else LGTM

File: internal/compose/helm.go [hunk 2]
```diff
@@ -117,6 +117,10 @@ func installHelmRelease(name string, release *cfg.Release) {
 		args = append(args, "--skip-crds")
 	}
 
+	if release.Wait {
+		args = append(args, "--wait")
+	}
+
```
Comment (tuunit) [inline internal/compose/helm.go:123]: One more thing, can you move this block down below the `Timeout` block to keep the order in line with the struct fields

**Review by tuunit — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dberuben) [reply CHANGELOG.md:3]: thank you

File: internal/compose/helm.go (hunk 2 above)
Comment (dberuben) [reply internal/compose/helm.go:123]: Yeah make more sense 👍

**Review by tuunit — APPROVED** [review]

**Pull request merged**